dashboard/db/migrate/20161111020106_create_pd_enrollment_notifications.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePdEnrollmentNotifications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_enrollment_notifications do |t|
+      t.timestamps
+      t.belongs_to :pd_enrollment, null: false
+
+      # The name of notification: i.e. name of action in Pd::WorkshopMailer
+      t.string :name
```

Comment:
Oh nice. I didn't know about `comment:`. Thanks.

re: `action_name` or `notification_name`, do you think that's more clear? I chose against that initially since the table is already called enrollment_notifications, so I'd assume the name is the notification name for an enrollment. Even with that I would want the comment for more context.